webpack.common.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const StyleLintPlugin = require('stylelint-webpack-plugin')
+const WebpackPluginCopy = require('webpack-plugin-copy')
+
+module.exports = {
+  entry: './src/main.js',
+  output: {
+    filename: './bundle.js',
+    path: path.resolve(__dirname, 'public')
+  },
+  module: {
+    rules: [
+      {
+        enforce: 'pre',
+        test: /\.js$/,
+        loader: 'standard-loader',
+        exclude: /(node_modules)/,
+        options: {
+          error: false,
+          snazzy: true
+        }
+      },
+      {
+        test: /\.js$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['react']
+          }
+        }
+      }
+    ]
+  },
+  plugins: [
+    new CleanWebpackPlugin(['public']),
```

Comment:
You can probably move this plugin to the prod config since this is not needed for development.